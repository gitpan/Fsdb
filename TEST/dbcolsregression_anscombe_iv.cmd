prog='dbcolsregression'
args='iv_x iv_y'
cmp='diff -cb '
in=TEST/dbcolsregression_anscombe.in

