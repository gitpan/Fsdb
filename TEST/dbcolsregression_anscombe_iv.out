#fsdb slope intercept confcoeff n
# Anscombe's Quartet
# from F.J. Anscombe, "Graphs in Statistical Analysis," American Statistician, 27 (February 1973), 17-21.
# via http://en.wikipedia.org/wiki/Anscombe%27s_quartet
0.49991 3.0017 0.81652 11
#   | dbcolsregression iv_x iv_y
